projects/izumi-iziswap/index.js

```diff
@@ -27,40 +30,48 @@ const blacklistedTokens = [
 
 const tvl = async (_, _1, _2, { api }) => {
   const chain = api.chain
-  const toa = []
-  let i = 1
-  let foundLastPool = false
+  const toa = [] 
   const chunkSize = 10
-  const poolMetaData = []
   const bTokens = [...blacklistedTokens]
+  const allPools = []
+  const allPoolMetas = []
 
-  do {
-    const calls = []
-    for (let j = i; j < i + chunkSize; j++)
-      calls.push(j)
-    i += chunkSize
-    const poolMetas = await api.multiCall({
-      target: poolHelpers[chain],
-      abi: abi.poolMetas,
-      calls,
-    })
-    for (const output of poolMetas) {
-      if (output.tokenX === nullAddress && output.fee === '0') {
-        foundLastPool = true
-        break;
+  for(const manager of poolHelpers[chain]) {
+    console.info(manager)
```

Comment:
code looks good, remove remove the console line

Reply:
Sorry, my miss. Code updated.